### ISSUE TITLE
4X4 Evo (NTSC)

### ISSUE THREAD

**Issue by delon5** [Rinnegatamante/flycast-vita-compatibility-list#168]
Powered by windows CE
![998C4FB7-94A3-4AEC-BED5-8817279DA295](https://user-images.githubusercontent.com/25923990/186780242-ab672dae-b736-4bb1-bfe5-18a9d6c6d706.jpeg)

didn’t make it past this screen in the first few seconds.

![5DF3F090-8742-477D-AFD5-CE985AAEC84E](https://user-images.githubusercontent.com/25923990/186780264-436c6e50-bdff-43ac-9cb7-13a13ece5968.jpeg)

Crash

**Comment by Rinnegatamante** [comment]
Did you try to change Self Modifying Code Checks option?

**Comment by delon5** [comment]
Yeah tried both reduced and full. It doesn’t explicitly crash but just stays on black screen. Have waited on screen for 20mins to see if it will get in game. It doesn’t.

**Comment by Rinnegatamante** [comment]
If you press select and resume, does the game still stay full black?

**Comment by delon5** [comment]
If self modifying check’s are off it presents the face image above and stays on that. If I click select and resume it freezes and doesn’t return to game. I can still press select again to exit so the emu doesn’t crash.

With reduced on just a black screen and the same behavior cannot return to the game after pressing select.
